src/lib/routes/admin-api/client-metrics.ts

```diff
@@ -18,39 +32,79 @@ class ClientMetricsController extends Controller {
         config: IUnleashConfig,
         {
             clientMetricsServiceV2,
-        }: Pick<IUnleashServices, 'clientMetricsServiceV2'>,
+            openApiService,
+        }: Pick<IUnleashServices, 'clientMetricsServiceV2' | 'openApiService'>,
     ) {
         super(config);
         this.logger = config.getLogger('/admin-api/client-metrics.ts');
 
         this.metrics = clientMetricsServiceV2;
+        this.openApiService = openApiService;
+
+        this.route({
+            method: 'get',
+            path: '/features/:name/raw',
+            handler: this.getRawToggleMetrics,
+            permission: NONE,
+            middleware: [
+                openApiService.validPath({
+                    operationId: 'getRawToggleMetrics',
+                    tags: ['admin'],
+                    responses: {
+                        200: createResponseSchema('featureMetricsSchema'),
+                    },
+                }),
+            ],
+        });
 
-        this.get('/features/:name/raw', this.getRawToggleMetrics);
-        this.get('/features/:name', this.getToggleMetricsSummary);
+        this.route({
+            method: 'get',
+            path: '/features/:name',
+            handler: this.getToggleMetricsSummary,
+            permission: NONE,
+            middleware: [
+                openApiService.validPath({
+                    operationId: 'getToggleMetricsSummary',
```

Comment:
Could rename this handler and operationId to `getFeatureMetrics` to match the other one.

Reply:
Went with getFeatureUsageSummary